cmd/csi-node-driver-registrar/main.go

```diff
@@ -115,12 +116,15 @@ func main() {
 		klog.Warning("--connection-timeout is deprecated and will have no effect")
 	}
 
+	// Unused metrics manager, necessary for connection.Connect below
+	cmm := metrics.NewCSIMetricsManagerForSidecar("")
```

Comment:
I suppose it wasn't made optional because all sidecars are expected to actually server metrics data.